packages/expo-tracking-transparency/ios/TrackingTransparencyModule.swift

```diff
@@ -16,6 +16,7 @@ public class TrackingTransparencyModule: Module {
         reject: promise.legacyRejecter
       )
     }
+    .runOnQueue(.main)
```

Comment:
The crash that you provided is coming from `requestPermissionsAsync`, is it also the case in `getPermissionsAsync`? If not, I think it would be better not to run it on the main thread.

Reply:
I'm not sure. I've only seen this crash twice (9 days apart - different users), so it seems to be somewhat rare. Whether `getPermissionsAsync` is safe to call off the main thread is up to apple's implementation. My guess is that it's not required to call on the main thread, but this seemed safer. I'll let you make that decision though.
I'm guessing it's only `requestPermissionsAsync` that's required to be on the main thread, because it shows UI, whereas `getPermissionsAsync` won't result in UI showing, but I can't know for sure (without reverse engineering apple's implementation. Or them updating the documentation.)